status-computation/java/src/main/java/sync/Downtimes.java

```diff
@@ -58,6 +58,10 @@ public int insert(String hostname, String service, String startTime,  String end
     }
     
     public ArrayList<String> getPeriod(String hostname, String service){
+    	//Fix for specific service SRMv2 which actually reports as SRM
+    	if (service.equals("SRMv2")) {
+    		service = "SRM";
+    	}
```

Comment:
I thought that we do not need to have this mapping in the compute engine any more